src/modules/routes/RouteData.ts

```diff
@@ -160,7 +157,10 @@ Routes.add(new RegionRoute(
     new RoutePokemon({
         land: ['Pidgey', 'Ekans', 'Sandshrew', 'Vulpix', 'Meowth', 'Growlithe'],
     }),
-    [new RouteKillRequirement(10, Region.kanto, 10)],
+    [
+        new RouteKillRequirement(10, Region.kanto, 10),
+        new ClearDungeonRequirement(1, getDungeonIndex('Rock Tunnel')),
+    ],
```

Comment:
```suggestion
    [new ClearDungeonRequirement(1, getDungeonIndex('Rock Tunnel'))],
```
If route 10 is needed to access Rock Tunnel, which is required to access Lavender Town, then the requirement for route 10 here isn't needed.